app/src/main/java/com/hedvig/app/SplashActivity.kt

```diff
@@ -85,6 +103,50 @@ class SplashActivity : BaseActivity() {
         } ?: startDefaultActivity(loginStatus)
     }
 
+    private fun startBirthdayActivity(){
+        startActivity(Intent(this, BirthdayActivity::class.java))
+    }
+
+    private fun isBirthday(): Boolean {
+        val socialSecurityNumber = "199701174617"
+
+        val birthMonth = socialSecurityNumber.substring(4, socialSecurityNumber.length - 6).toInt()
```

Comment:
You have the same logic in `BirthdayViewModel.kt` if you need to do this in two places please have a look at kotlin extensions and maybe create one.